lib/ui/list/infinite_list.dart

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Ok, i see that this has been commented out... did something break because of this? I personally wanted to make a generic infinite list so I could repurpose the code for like search results and comments.... but like if its not convention then ig we can remove it.

Reply:
I commented it out bc mainly bc I was just trying to find a quick way to implement it for the posts list view, I felt like the infinite list only made things more complicated